src/modules/user/user.service.ts

```diff
@@ -13,11 +13,11 @@ export default class UserService {
     private userRepository: Repository<User>
   ) {}
 
-  async createUser(user: CreateNewUserOptions) {
+  async createUser(user: CreateNewUserOptions): Promise<any> {
     const newUser = new User();
     Object.assign(newUser, user);
     newUser.is_active = true;
-    await this.userRepository.save(newUser);
+    return await this.userRepository.save(newUser);
```

Comment:
No need for await keyword when returning a promise from function. Reduces response time needlessly